store/sqlstore/store.go

```diff
@@ -1678,12 +1669,74 @@ func IsDuplicate(err error) bool {
 	return false
 }
 
-// VersionString converts an integer representation of a DB version
+// ensureMinimumDBVersion gets the DB version and ensures it is
+// above the required minimum version requirements.
+func (ss *SqlStore) ensureMinimumDBVersion() (bool, error) {
+	ver, err := ss.GetDbVersion(true)
+	if err != nil {
+		return false, err
+	}
```

Comment:
I am thinking that if we decouple this from getting the current db version, we can then more easily test this method in isolation since it's doing a bit of parsing as well which would be good to get covered.

Reply:
Sure.